src/intl/de.js

```diff
@@ -155,6 +155,7 @@ export default {
     <li><kbd>p</kbd> das Profil des Verfassers öffnen</li>
     <li><kbd>l</kbd> den Link des aktuellen Tröts in einem neuen Tab öffnen</li>
     <li><kbd>x</kbd> den Text hinter der Inhaltswarnung anzeigen oder verbergen</li>
+    <li><kbd>z</kbd> den Text hinter der Inhaltswarnung anzeigen oder verbergen (alle)</li>
```

Comment:
I don't speak German but I'm making an effort here. I don't know how to keep translations up-to-date long-term.